### PR TITLE
Updating Demystifying Istio's Sidecar Injection Model Page

### LINKED ISSUE #4509: Automatic injection demystification page incorrect
I am concerned about the text in https://preliminary.istio.io/blog/2019/data-plane-setup/#automatic-injection

It says

```
In case you want to change the default behavior, like the namespaces where Istio applies the injection, you can edit the MutatingWebhookConfiguration and restart the sidecar injector pod.

The automatic sidecar injection depends on the namespaceSelector webhook but also on the default injection policy and the per-pod override annotation.
```

Users don't need to restart the injector when they wish to change the namespaces where Istio applies injection.  (They might need to restart the pod if they wish to shoot themselves in the foot with a custom and non-standard injection label scheme.)

There is no such thing as a "namespaceSelector webhook".  The sidecar injection depends on "the `_istio-sidecar-injector_ MutatingWebhookConfiguration".

### PR DESCRIPTION
**Pull request by chugtum**
Fixes #4509

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[X ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by chugtum** [comment]
> A couple of spell check issues, but otherwise lgtm.

Thanks @frankbu. Since I don't have write access to the repo, what's the next step to get this merged ?

File: content/blog/2019/data-plane-setup/index.md [hunk 1]
```diff
@@ -260,9 +260,9 @@ $ kubectl get svc --namespace=istio-system | grep sidecar-injector
 istio-sidecar-injector   ClusterIP   10.102.70.184   <none>        443/TCP             24d
 {{< /text >}}
 
-This configuration ultimately does pretty much the same as we saw in manual injection. Just that it is done automatically during pod creation, so you won’t see the change in the deployment. You need to use `kubectl describe` to see the sidecar proxy and the init proxy. In case you want to change the default behavior, like the namespaces where Istio applies the injection, you can edit the `MutatingWebhookConfiguration` and restart the sidecar injector pod.
+This configuration ultimately does pretty much the same as we saw in manual injection. Just that it is done automatically during pod creation, so you won’t see the change in the deployment. You need to use `kubectl describe` to see the sidecar proxy and the init proxy. 
 
-The automatic sidecar injection depends on the `namespaceSelector` webhook but also on the default injection policy and the per-pod override annotation.
+The automatic sidecar injection not only depends on the webhoook's namespaceSelector mechanism but also on the default injection policy and the per-pod override annotation.
```
Comment (frankbu) [inline content/blog/2019/data-plane-setup/index.md]:
```suggestion
The automatic sidecar injection not only depends on the `namespaceSelector` mechanism of the webhook, but also on the default injection policy and the per-pod override annotation.
```

Comment (frankbu) [reply content/blog/2019/data-plane-setup/index.md]: ^^^ My suggestion for avoiding the spelling errors

**Comment by chugtum** [comment]
> > A couple of spell check issues, but otherwise lgtm.
> 
> Thanks @frankbu. Since I don't have write access to the repo, is there any action I need to take to get this merged ?

Thanks @frankbu. Have accepted your suggestion and committed. Hope all is good now to have it merged ?

**Comment by chugtum** [comment]
@frankbu  Just realized that since I accepted your suggestion/committed from GitHub itself, and as it added you as a co-author, the CLA was set to No. Hence whoever is merging, would need to force set it to YES manually before merging.

**Pull request merged**